lib/voicebase/v2/client.rb

```diff
@@ -26,25 +26,44 @@ def authenticate!
       end
 
       def upload_media(args = {}, headers = {})
+
+        media_url = require_media_file_or_url(args)
+
+        # test URL from VoiceBase
```

Comment:
Do you still want to keep this test URL in there? (It's fine if so, just wanted to check)